src/ui/networkpage.rs

```diff
@@ -207,13 +216,36 @@ fn details_table_from_iface<'a, 'b>(iface: &'a NetworkInterfaceStatus) -> Vec<Ro
         NetworkType::Cellular(_) => {}
     }
 
+    // Row 5: Errors if any. One line per error. The heights of the rows are set to the number of errors
+    let errors = iface.errors.as_ref().map_or_else(
+        || "N/A".to_string(),
+        |list| {
+            list.iter()
+                .map(|ip| ip.to_string())
+                .collect::<Vec<_>>()
+                .join("\n")
+        },
+    );
+    let errors_row_height = iface.errors.as_ref().map_or(1, |v| v.len());
+    let errors_row = Row::new(vec![
+        Cell::from("Errors").style(Style::new().yellow()),
+        Cell::from(errors).style(Style::new().white()),
+    ]);
+    table.push(errors_row.height(errors_row_height as u16));
+
     table
 }
 
 impl IPresenter for NetworkPage {
     fn render(&mut self, area: &Rect, frame: &mut Frame<'_>, model: &Rc<Model>, _focused: bool) {
-        let estimated_width =
-            IFACE_LABEL_LENGTH + LINK_STATE_LENGTH + IPV6_AVERAGE_LENGTH + MAC_LENGTH + 3 + 2 + 2; // for spacers and borders and selector
+        let estimated_width = IFACE_LABEL_LENGTH
+            + LINK_STATE_LENGTH
+            + IPV6_AVERAGE_LENGTH
+            + MAC_LENGTH
+            + CTRL_STATUS_LENGTH
+            + 3
+            + 2
+            + 2; // for spacers and borders and selector
```

Comment:
Can we use constants instead of magic numbers? Really appreciate comments tho

Reply:
@uncleDecart we should, yes